src/Frontend/Fragment/Comment.hs

```diff
@@ -22,10 +22,9 @@ import qualified Frontend.Path as U
 
 
 data CommentWidget = CommentWidget
-    { _cwRenderContext :: RenderContext
-    , _cwIdeaCaps      :: [Capability]
-    , _cwComment       :: Comment
-    , _cwPhase         :: Phase
+    { _cwCapCtx   :: CapCtx
+    , _cwIdeaCaps :: [Capability]
```

Comment:
is this worth a `FIXME`?  i would have thought that caps are implicit in capctx.  if this is just to avoid redundant computations, there should be a comment saying so.